nav2-usability/scenario-code/deeplink-pathparam/deeplink_pathparam_vrouter.dart

```diff
@@ -0,0 +1,124 @@
+// Copyright 2021, the Flutter project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// Deeplink path parameters example
+/// Done using VRouter
+
+import 'package:flutter/material.dart';
+import 'package:vrouter/vrouter.dart';
+
+void main() {
+  runApp(BooksApp());
+}
+
+class Book {
+  final String title;
+  final String author;
+
+  Book(this.title, this.author);
+}
+
+class BooksApp extends StatelessWidget {
+  final List<Book> books = [
+    Book('Stranger in a Strange Land', 'Robert A. Heinlein'),
+    Book('Foundation', 'Isaac Asimov'),
+    Book('Fahrenheit 451', 'Ray Bradbury'),
+  ];
+
+  @override
+  Widget build(BuildContext context) {
+    return VRouter(
+      routes: [
+        VWidget(
+          path: '/',
+          widget: BooksListScreen(books: books),
+          stackedRoutes: [
+            stackedRoutes: [
```

Comment:
remove one of these?

Reply:
Done in 458e1b46d3e9537077b737405e4e19a8fc46d849, don't know what happened there.